tests/servers/test_basehttp.py

```diff
@@ -15,6 +15,13 @@ def sendall(self, data):
         self.makefile("wb").write(data)
 
 
+class UnclosableBytesIO(BytesIO):
+    def close(self):
+        # WSGIRequestHandler closes the output file; we need to make this a no-op
+        # so we can still read its contents.
```

Comment:
Wrap comments at 79 chars.

Reply:
I have confused myself, do the spaces at the beginning count in the char count?

Reply:
> I have confused myself, do the spaces at the beginning count in the char count?

Yes, so this is 81 chars now.